pytest_catchlog.py

```diff
@@ -154,6 +168,48 @@ def pytest_runtest_teardown(self, item):
         with self._runtest_for(item, 'teardown'):
             yield
 
+    def configure_console_handler(self, verbosity):
+        # Prepare the handled_levels dictionary
+        log_levels = []
+        handled_levels = {}
+        try:
+            available_levels = logging._levelNames.keys()
+        except AttributeError:
+            # Python >= 3.4
+            # And also PyPy3-2.4.0(Python 3.2) apparently...
+            # https://travis-ci.org/eisensheng/pytest-catchlog/jobs/91417631
+            available_levels = logging._levelToName.keys()
```

Comment:
I have another approach, add a hook and let the user define additional logging levels, which, I believe is the most future proof way of doing this. We add a mapping of the log levels that we handle, and, if no other logging levels are defined.....